src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -190,17 +190,28 @@
             }
         });
     };
-    const addField = ({ targetContainer, draggedItemPosition }) => {
+    const addField = ({ targetContainer, draggedItemPosition, fieldRendered }) => {
         if (!sourceContainer.classList.contains('ibexa-available-field-types__list')) {
-            insertFieldDefinitionNode(currentDraggedItem, { targetContainer, draggedItemPosition });
+            const insertedField = insertFieldDefinitionNode(currentDraggedItem, { targetContainer, draggedItemPosition });
+
+            insertedField.classList.add('ibexa-collapse--field-definition-loading');
+
             afterChangeGroup();
 
+            global.setTimeout(() => {
+                insertedField.classList.remove('ibexa-collapse--field-definition-loading');
+
+                removeHighlight(insertedField);
+            }, TIMEOUT_REMOVE_HIGHLIGHT);
+
             return;
         }
 
         const { languageCode } = sectionsNode.dataset;
         const { itemIdentifier } = currentDraggedItem.dataset;
         const { fieldsGroupId } = targetContainer.closest('.ibexa-collapse--field-definitions-group').dataset;
+        const loadingField = createFieldDefinitionNode(fieldRendered, { targetContainer, draggedItemPosition });
+        let addedField = loadingField;
```

Comment:
New variable was added as DOM field changes in lifecycle. This variable usage is only to be passed to removeHighlight.

After dropping element "fake" DOM element is created (which has highlight, so it has to be passed to removeHighlight) and request is send to backend. When it receive response, this fake DOM element is removed and replaced with new DOM element with html from backend, so argument passed to removeHighlight has to be changed as well.